packages/console/src/hooks/use-connector-in-use.ts

```diff
@@ -23,12 +23,14 @@ const useConnectorInUse = () => {
       const relatedIdentifier =
         type === ConnectorType.Email ? SignInIdentifier.Email : SignInIdentifier.Sms;
 
-      return (
-        data.signIn.methods.some(
-          ({ identifier, verificationCode }) => verificationCode && identifier === relatedIdentifier
-        ) ||
-        (data.signUp.identifiers.includes(relatedIdentifier) && data.signUp.verify)
+      const usedInSignUp =
+        data.signUp.identifiers.includes(relatedIdentifier) && data.signUp.verify;
+
+      const usedInSignIn = data.signIn.methods.some(
+        ({ identifier, verificationCode }) => verificationCode && identifier === relatedIdentifier
       );
+
+      return usedInSignUp || usedInSignIn;
```

Comment:
If the method is used as a SignUp method, it is also required as a SignIn Method.  Is this redundant?

Reply:
No, we allow the following config:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/10806653/210484099-4d142095-3a64-48ca-b70d-b4c68c750dc7.png">

Reply:
Discussed offline